dataservice/api/sample_relationship/models.py

```diff
@@ -32,18 +32,23 @@ class SampleRelationship(db.Model, Base):
     parent_id = db.Column(
         KfId(),
         db.ForeignKey('sample.kf_id'),
-        nullable=False,
+        nullable=True,
         doc='kf_id of one sample in the relationship')
 
     child_id = db.Column(
         KfId(),
         db.ForeignKey('sample.kf_id'),
-        nullable=False,
+        nullable=True,
```

Comment:
keep this false, I think

Reply:
I wasn't sure what to do about this. I thought maybe we make this nullable too bc maybe there is a time when a parent sample gets added in but it doesn't have children yet? Idk if that is realistic but that was the motivation